posts/2015-01-09-Rust-1.0-alpha.md

```diff
@@ -92,17 +92,17 @@ A detailed list of possible breaking changes to *stable* features/APIs can be fo
 
 * **Improvements to [associated types](https://github.com/rust-lang/rust/issues/17307) and [unboxed closures](https://github.com/rust-lang/rust/issues/18101):** Both of these features are functional, but we plan to make significant ergonomic improvements during the beta cycle, such as [more sophisticated capture-clause inference](https://github.com/rust-lang/rfcs/blob/master/text/0231-upvar-capture-inference.md) and more [uniform support for shorthands like `X::Type`](https://github.com/rust-lang/rust/issues/20300).
 
-* **Improvements to generics:** We will explore [avenues](http://smallcultfollowing.com/babysteps/blog/2014/07/06/implied-bounds/) to cut down on the number of redundant trait bounds that are currently required.
+* **Improvements to generics:** We will explore [avenues](https://smallcultfollowing.com/babysteps/blog/2014/07/06/implied-bounds/) to cut down on the number of redundant trait bounds that are currently required.
 
-* **Path reform:** The `path` module will soon be [reformed](https://github.com/rust-lang/rfcs/pull/474) to make use of DST, to give a better account of platform differences, and to more closely align with the [new C++ standard](http://article.gmane.org/gmane.comp.lib.boost.devel/256220). Implementation work is largely complete, and the rollout should occur soon after alpha.
+* **Path reform:** The `path` module will soon be [reformed](https://github.com/rust-lang/rfcs/pull/474) to make use of DST, to give a better account of platform differences, and to more closely align with the [new C++ standard](https://ghostarchive.org/archive/cJo6w). Implementation work is largely complete, and the rollout should occur soon after alpha.
```

Comment:
could we use the wayback machine for these? I've never heard of ghostarchive before, and the IA is very likely to stay around far into the future.

Reply:
I think both sites are likely to survive in the foreseeable future, and the first time i heard about Rust it was already 6 years in its existence.

I did the change both links to wayback, per your request. Because you would know what is good, after all you helped make my favorite programming language of all time.

BTW, i appreciate your work with Rust. You are one of my most favorite celebrity programmers. Keep up the good work.

Reply:
Thank you! Sorry it took me so long to initially review.